Makefile

```diff
@@ -195,10 +195,12 @@ ifeq ($(dbbackend),postgres)
 	docker rm lnd-postgres --force || echo "Starting new postgres container"
 
 	# Start a fresh postgres instance. Allow a maximum of 500 connections so
-	# that multiple lnd instances with a maximum number of connections of 50
-	# each can run concurrently.
-	docker run --name lnd-postgres -e POSTGRES_PASSWORD=postgres -p 6432:5432 -d postgres:13-alpine -N 500
-	docker logs -f lnd-postgres &
+	# that multiple lnd instances with a maximum number of connections of 20
+	# each can run concurrently. Note that many of the settings here are
+	# specifically for integration testing and are not fit for running
+	# production nodes.
+	docker run --name lnd-postgres -e POSTGRES_PASSWORD=postgres -p 6432:5432 -d postgres:13-alpine -N 500 -c max_pred_locks_per_transaction=1024 -c max_locks_per_transaction=128 -c jit=off -c work_mem=8MB -c checkpoint_timeout=10min -c enable_seqscan=off
+	docker logs -f lnd-postgres >itest/postgres-log 2>&1 &
```

Comment:
can we name this `postgres.log` instead and add it to the gitignore

Reply:
cool yeah let's fix it when we move DB instance per tranche, just FYI we usually do logs cleanup here,
https://github.com/lightningnetwork/lnd/blob/c8cfa59316d827377ba80b7cd1295f678068baf1/lntest/node/harness_node.go#L669-L674